v4m-container/Dockerfile

```diff
@@ -1,13 +1,13 @@
-
-# Version of the Viya Monitoring for Kubernetes deployment to download to the container
-ARG V4M_TAG="stable"
-
 ARG ALPINE_TAG=latest
 
 # Need glibc to support the OpenShift 'oc' CLI
 # https://hub.docker.com/r/frolvlad/alpine-glibc/
 FROM frolvlad/alpine-glibc:$ALPINE_TAG
 
+# Version of the Viya Monitoring for Kubernetes deployment to download to the container
+ARG V4M_TAG="stable"
```

Comment:
This was being ignored because it was before the `FROM`, which essentially resets the ARG, so it was being treated as blank.